cadasta/plugin.py

```diff
@@ -201,26 +192,51 @@ def _create_options_wizard_action(self):
             callback=self.show_options_wizard
         )
 
-    def show_options_wizard(self):
-        """Show the options wizard."""
-        dialog = CadastaLogin()
-        dialog.show()
-        dialog.exec_()
+    def _create_project_creation_wizard(self):
+        """Create action for project creation wizard."""
+        icon_path = ':/plugins/cadasta-qgis-plugin/icon.png'
```

Comment:
@meomancer You should use `resources_path('images', 'white_icon.png')`. This function will take care of the real OS separator.